pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs

```diff
@@ -343,6 +344,16 @@ function actionButtonFormatter(_value, row, index) {
         .append('Grade &amp; Close')
         .appendTo(menu);
 
+    $('<a>')
```

Comment:
I know you're just matching existing code, but we should really just use a `<button>` here instead of jumping through all the hoops of `<a role="button" tabindex=0">`.